src/main/java/org/cryptomator/common/settings/Settings.java

```diff
@@ -75,6 +75,7 @@ public class Settings {
 	public final BooleanProperty checkForUpdates;
 	public final ObjectProperty<Instant> lastUpdateCheckReminder;
 	public final ObjectProperty<Instant> lastSuccessfulUpdateCheck;
+	public final StringProperty previouslyUsedVaultDirectory;
```

Comment:
For consistency reasons, use an `ObjectProperty<Path>`, similar to `VaultSettings.mountPoint`, doing all the String ↔︎ Path conversion here instead of in the business logic.